addons/binding/org.openhab.binding.nest/src/main/java/org/openhab/binding/nest/internal/data/Structure.java

```diff
@@ -0,0 +1,187 @@
+package org.openhab.binding.nest.internal.data;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+import org.openhab.binding.nest.internal.data.SmokeDetector.AlarmState;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * The structure details from nest.
```

Comment:
Please add a JavaDoc with `@author` tag containing a real name of the author. See [Code Style](http://docs.openhab.org/developers/development/guidelines.html#a-code-style).